### PR TITLE
[#462] fix: adjust library name for publishing

### PR DESCRIPTION
**Pull request by krzema12**
A follow-up after the commit with major rename.

### CODE REVIEW

**Comment by jmfayard** [comment]
To check that things are right, I uncommented your variables like this



So far so good for me

File: buildSrc/src/main/kotlin/buildsrc/convention/dsl-publishing.gradle.kts [hunk 1]
```diff
@@ -29,9 +30,9 @@ publishing {
...
                 scm {
-                    connection.set("scm:git:git://github.com/$githubUser/$libraryName.git/")
-                    developerConnection.set("scm:git:ssh://github.com:$githubUser/$libraryName.git")
-                    url.set("https://github.com/$githubUser/$libraryName.git")
+                    connection.set("scm:git:git://github.com/$githubUser/$repositoryName.git/")
+                    developerConnection.set("scm:git:ssh://github.com:$githubUser/$repositoryName.git")
+                    url.set("https://github.com/$githubUser/$repositoryName.git")
                 }
 
                 developers {
```
Comment (jmfayard) [inline buildSrc/src/main/kotlin/buildsrc/convention/dsl-publishing.gradle.kts:38]: nitpick: you can use `githubUser` at line 40

**Comment by jmfayard** [comment]
Then I run `./gradle publishToMavenLocal`

I wanted to check the generated `pom.xml` inside the `build`  folder
Unfortunately it fails me with the error [Cannot perform signing task ':library:signMavenJavaPublication' because it has no configured signatory](https://scans.gradle.com/s/2qlnfs3jso354/console-log?anchor=50&page=1)

That's the next thing I would do. 
Run the task and triple check the `pom.xml`

**Review by jmfayard — DISMISSED** [review]

**Comment by krzema12** [comment]
@jmfayard thanks for the idea how to test it - I'll compare POMs, before the whole rename and after this PR.
FYI, you can skip signing this way:
https://github.com/krzema12/github-workflows-kt/blob/1885608c400a9212f84760bad63f89512629b3c2/jitpack.yml#L6

**Comment by krzema12** [comment]
```diff
diff --git a/before.pom b/after.pom
index da2c249..9a1ef0e 100644
--- a/before.pom
+++ b/after.pom
@@ -11,7 +11,7 @@
   <version>0.28.0</version>
   <name>github-actions-kotlin-dsl</name>
   <description>Authoring GitHub Actions workflows in Kotlin.</description>
-  <url>https://github.com/krzema12/github-actions-kotlin-dsl</url>
+  <url>https://github.com/krzema12/github-workflows-kt</url>
   <licenses>
     <license>
       <name>Apache License, version 2.0</name>
@@ -26,9 +26,9 @@
     </developer>
   </developers>
   <scm>
-    <connection>scm:git:git://github.com/krzema12/github-actions-kotlin-dsl.git/</connection>
-    <developerConnection>scm:git:ssh://github.com:krzema12/github-actions-kotlin-dsl.git</developerConnection>
-    <url>https://github.com/krzema12/github-actions-kotlin-dsl.git</url>
+    <connection>scm:git:git://github.com/krzema12/github-workflows-kt.git/</connection>
+    <developerConnection>scm:git:ssh://github.com:krzema12/github-workflows-kt.git</developerConnection>
+    <url>https://github.com/krzema12/github-workflows-kt.git</url>
   </scm>
   <dependencies>
     <dependency>

```

**Pull request merged**